curriculum/challenges/english/10-coding-interview-prep/rosetta-code/euler-method.english.md

```diff
@@ -68,18 +68,20 @@ and to compare with the analytical solution.
 </ul> 
 </section>
 
+First parameter to the function is initial time, second parameter is initial temperature, third parameter is elapsed time and fourth parameter is step size.
```

Comment:
Was this intended to be a part of the challenge instructions? As it stands, this isn't displayed on the challenge page.

Reply:
@Manish-Giri Thanks for catching that!

@nibble0101 Make sure to add the text inside the closing `section` tag.